applications/pelion_client/src/modules/pelion_trace.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "mbed-trace/mbed_trace.h"
+#include "mbed-trace-helper.h"
+
+#define MODULE pelion_trace
+#include <caf/events/module_state_event.h>
+
+
+static void init_pelion_trace(void)
+{
+	bool ret;
+#ifndef MCC_MINIMAL
```

Comment:
Can we see `MCC_MINIMAL`?

Reply:
Cleaned - basically it is a leftover from code sample from mbed. Removed.